antlr_config/ManimParser.g4

```diff
@@ -4,13 +4,20 @@ options {
   tokenVocab=ManimLexer;
 }
 
-program: (stat SEMI)+ EOF;
+program: function* (stat SEMI)+ EOF;
+
+function: DEF IDENT OPEN_PARENTHESIS param_list? CLOSE_PARENTHESIS (COLON type)? OPEN_CURLY_BRACKET (stat SEMI)+ CLOSE_CURLY_BRACKET;
```

Comment:
Is having an empty function syntactically incorrect in our language then?

Reply:
right no yes I can change it if we want to allow empty functions?

Reply:
Yeah I think it should be allowed